lib/main_game/melvyn_plus_plus_game.dart

```diff
@@ -30,17 +29,15 @@ class MelvynPlusPlusGame extends FlameGame
     );
     unawaited(add(tiledMap));
 
-    final Paint knobPaint = BasicPalette.white.withAlpha(200).paint();
-    final Paint backgroundPaint = BasicPalette.white.withAlpha(100).paint();
-    joystick = JoystickComponent(
-      knob: CircleComponent(radius: 20, paint: knobPaint),
-      background: CircleComponent(radius: 50, paint: backgroundPaint),
-      margin: const EdgeInsets.only(left: 40, bottom: 40),
-    );
+    joystick = Joystick();
 
     player = Player(joystick);
-    unawaited(add(player));
+    final ActionButtons buttons = ActionButtons(player: player);
+    await buttons.initialize();
 
+    unawaited(add(player));
```

Comment:
Ça serait cool si on pouvait faire un future await car si l'un des trucs ne s'affiche pas c'est problématique

Ptete un try and catch ?

Reply:
Done!